charts/k0s/templates/coredns.yaml

```diff
@@ -73,7 +73,7 @@ data:
             errors
             health
             ready
-            kubernetes cluster.local in-addr.arpa ip6.arpa {
+            kubernetes {{- if .Values.vcluster.clusterDomain }} {{ .Values.vcluster.clusterDomain }} {{- end }} cluster.local in-addr.arpa ip6.arpa {
```

Comment:
Why is the `-` needed here in `{{-`, wouldn't `{{ if ` be better here?

Reply:
yep, from the template doc: [`{{- (with the dash and space added) indicates that whitespace should be chomped left, while -}} means whitespace to the right should be consumed.`](https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace). i've tested the output with `helm template` and spacing looked correct for both cases